packages/sp1-ics07-tendermint-prover/Cargo.toml

```diff
@@ -13,13 +13,11 @@ ibc-eureka-solidity-types = { workspace = true }
 
 alloy-sol-types = { workspace = true }
 
-bincode    = { workspace = true }
-serde_cbor = { workspace = true, features = ["std"] }
+prost      = { workspace = true }
```

Comment:
I decided that we either use protobuf encoding (convenient for ibc types) or ethereum abi encoding (convenient for contracts). There is no "clean" way to use just one of these in my opinion. And this keeps the code clean as well